### DISCUSSION TITLE
Donating for the free Version

### DISCUSSION THREAD

**Discussion by ghost** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#270] [General]
Hi there,
i'm using aldente free since the 1.0 release and i'm loving the app.
i'm rocking an old Late 2013 15" MBP (and btw, it's working nicely if you need a feedback for this machine since i don't find it mentioned anywhere) and tbh, even if i like the idea of a pro version the price it's a little out of my budget at the moment, but still i would love to be able to contribute to the app developing in some way, so i was wondering, would you be able to accept a little tip for your work? something like "buy us a spaghetti pack" LOL.
looking forward for a paypal.me link or something like that,
giorgio

**Comment by MatthiasKerbl** [comment]
Hi @gi-fo,

Thank you for your support!
We will include a possibility to donate on this Github page soon.

Kind regards,

Matthias
